README.md

```diff
@@ -198,6 +198,15 @@ NAME                     DESIRED   READY   NODE SELECTOR   AGE
 gpudeviceplugin-sample   1         1                       5s
 ```
 
+In order to limit the deployment to a specific device type,
+use one of kustomizations under deployments/operator/device.
+
+For example, to limit the deployment to FPGA, use:
+
+```bash
+$ kubectl apply -k deployments/operator/device/fpga
+```
+
```

Comment:
Should we explain here how to deploy operator with more than one enabled plugin?

Reply:
Thanks! Done.

Reply:
It's still unclear to me how to deploy more than one plugin. For example if I want to deploy operator with fpga and gpu support which path should I use for ```kubectl apply -k``` ? Where should I add ```--device fpga --device gpu``` command line options?

Reply:
OK, updated. Is this better?

Reply:
yep, much better, thanks.